Powershell scripts/VM Scanners/vmScannersOnboarding.ps1

```diff
@@ -0,0 +1,9 @@
+param(
+[Parameter(Mandatory = $True)] [string]$targetRgName,
+[Parameter(Mandatory = $False)] [hashtable]$exclusionTags=@{}
+)
+
+$objectid = (Get-AzADServicePrincipal -DisplayName "Microsoft Defender for Cloud Servers Scanner Resource Provider").id
+
+Register-AzResourceProvider -ProviderNamespace Microsoft.Security
+New-AzDeployment -Location "West Europe" -targetRgName $targetRgName -exclusionTags $exclusionTags -principalId $objectid -TemplateUri uriuriuri
```

Comment:
To be changed after the template will be approved

Reply:
Rename script file name to Onboard-VmScanners